pkg/kubelet/util/pluginwatcher/plugin_watcher.go

```diff
@@ -190,8 +202,10 @@ func (w *Watcher) traversePluginDir(dir string) error {
 
 		switch mode := info.Mode(); {
 		case mode.IsDir():
-			if err := w.fsWatcher.Add(path); err != nil {
-				return fmt.Errorf("failed to watch %s, err: %v", path, err)
+			if !w.containsBlacklistedDir(path) {
```

Comment:
if this is using fileutil.Walk, should we do this to avoid recursing into the ignored dir and registering watchers on paths we're just going to ignore?

```
case mode.IsDir():
  if w.containsBlacklistedDir(path) {
    return fileutil.SkipDir
  }
  ...
```

Reply:
Done